### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/manually.md`

### PR DESCRIPTION
**Pull request by xzmeng**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2352428947bde0f1b64fd8e362905079a77ad9f7 at: https://a2cb7b7a.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/manually.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+     ```console
+     $ pip install "uvicorn[standard]"
+
+     ---> 100%
+     ```
+
+     </div>
+
+     !!! tip
+         通过添加“标准”，Uvicorn 将安装并使用一些推荐的额外依赖项。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/manually.md:41]:
```suggestion

         通过添加`standard`，Uvicorn 将安装并使用一些推荐的额外依赖项。
```

File: docs/zh/docs/deployment/manually.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+     ```
+
+     </div>
+
+     ...或任何其他 ASGI 服务器。
+
+
+## 运行服务器程序
+
+您可以按照教程中的相同方式运行应用程序，但不使用`--reload`选项，例如：
```
Comment (shamosishen) [inline docs/zh/docs/deployment/manually.md:64]:
```suggestion
您可以按照之前教程中的相同方式运行应用程序，但不使用`--reload`选项，例如：
```

**Review by shamosishen — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4f7a3e058a3ff7376025a8eb4f1bb59ca88e830d at: https://6ca64508.fastapitiangolo.pages.dev

**Comment by shamosishen** [comment]
A tip is rendered incorrectly, can you check it out ?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5eb4e84b899a16361fa49dd3f8c3255e51deeb01 at: https://f8cffb41.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 085ac494dba19373e66677ebfff233024474d260 at: https://40693db6.fastapitiangolo.pages.dev

**Comment by xzmeng** [comment]
fixed

File: docs/zh/docs/deployment/manually.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+     `--reload` 选项消耗更多资源，并且更不稳定。
+
+     它在**开发**期间有很大帮助，但您**不应该**在**生产环境**中使用它。
+
+## Hypercorn with Trio
+
+Starlette 和 **FastAPI** 基于 <a href="https://anyio.readthedocs.io/en/stable/" class="external-link" target="_blank">AnyIO</a>， 所以它们才能同时与 Python 的标准库 <a href="https://docs.python.org/3/library/asyncio-task.html" class="external-link" target="_blank">asyncio</a> 和<a href="https://trio.readthedocs.io/en/stable/" class="external-link" target="_blank">Trio</a> 兼容。
+
+尽管如此，Uvicorn 目前仅与 asyncio 兼容，并且通常使用 <a href="https://github.com/MagicStack/uvloop" class="external-link" target="_blank">`uvloop`</a >,`asyncio`的高性能替代品。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/manually.md:102]:
```suggestion
尽管如此，Uvicorn 目前仅与 asyncio 兼容，并且通常使用 <a href="https://github.com/MagicStack/uvloop" class="external-link" target="_blank">`uvloop`</a >, 它是`asyncio`的高性能替代品。
```

**Review by shamosishen — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9a01d9248b695e780ff514f98cc7d6d536808e7d at: https://332759d7.fastapitiangolo.pages.dev

**Review by Aruelius — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you! @xzmeng 🍰 

And thanks for the reviews @shamosishen, @Aruelius! ☕

**Pull request merged**